.github/workflows/pull-request.yml

```
@@ -0,0 +1,354 @@
+  Check-Sensitive-Files:
+    if: ${{ github.actor != 'dependabot[bot]' && !contains(github.event.pull_request.labels.*.name, 'ignore-sensitive-files-pr') }}
+    name: Checks if sensitive files have been changed without authorization
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Fetch all history for all branches and tags
+      - name: Get Changed Unauthorized files
+        id: changed-unauth-files
+        run : | 
```

YAMLlint (1.37.1)

[error] too many spaces before colon

(colons)